mmv1/products/firebasehosting/CustomDomain.yaml

```diff
@@ -0,0 +1,514 @@
+      also grants that project ownership over `bar.foo.com`, unless you add
+      specific `TXT` records to `bar.foo.com` that grant a different project
+      ownership.
+     
+      If your `CustomDomain` is in an `OwnershipState` other than
+      `OWNERSHIP_ACTIVE` for more than 30 days and it hasn't been updated in at
+      least 30 days, Hosting's ownership systems delete the `CustomDomain`.
+
+      OWNERSHIP_MISSING:
+        Your `CustomDomain`'s domain name has no Hosting-related ownership records;
+        no Firebase project has permission to act on the domain name's behalf.
+
+      OWNERSHIP_UNREACHABLE:
+        Your `CustomDomain`'s domain name can't be reached. Hosting services' DNS
+        queries to find your domain name's ownership records resulted in errors.
+        See your `CustomDomain`'s `issues` field for more details.
+
+      OWNERSHIP_MISMATCH:
+        Your `CustomDomain`'s domain name is owned by another Firebase project.
+        Remove the conflicting `TXT` records and replace them with project-specific
+        records for your current Firebase project.
+
+      OWNERSHIP_CONFLICT:
+        Your `CustomDomain`'s domain name has conflicting `TXT` records that
+        indicate ownership by both your current Firebase project and another
+        project. Remove the other project's ownership records to grant the current
+        project ownership.
+
+      OWNERSHIP_PENDING:
+        Your `CustomDomain`'s DNS records are configured correctly. Hosting will
+        transfer ownership of your domain to this `CustomDomain` within 24 hours.
+
+      OWNERSHIP_ACTIVE:
+        Your `CustomDomain`'s domain name has `TXT` records that grant its project
+        permission to act on its behalf.
+    values:
+      - :OWNERSHIP_MISSING
+      - :OWNERSHIP_UNREACHABLE
+      - :OWNERSHIP_MISMATCH
+      - :OWNERSHIP_CONFLICT
+      - :OWNERSHIP_PENDING
+      - :ACTIVE
+  - !ruby/object:Api::Type::Enum
+    name: certPreference
+    default_from_api: true
+    description: |
+      A field that lets you specify which SSL certificate type Hosting creates
+      for your domain name. Spark plan `CustomDomain`s only have access to the
+      `GROUPED` cert type, while Blaze plan can select any option.
+    values:
+      - :GROUPED
+      - :PROJECT_GROUPED
+      - :DEDICATED
+  - !ruby/object:Api::Type::String
+    name: redirectTarget
+    description: |
+      A domain name that this CustomDomain should direct traffic towards. If
+      specified, Hosting will respond to requests against this CustomDomain
+      with an HTTP 301 code, and route traffic to the specified `redirect_target`
+      instead.
+  - !ruby/object:Api::Type::NestedObject
+    name: requiredDnsUpdates
+    output: true
+    description: |
+      A set of updates you should make to the domain name's DNS records to
+      let Hosting serve secure content on its behalf.
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: checkTime
```

Comment:
I don't believe the `output: true` behavior is inherited, so you may need to add it to each of the children here. Were you able to confirm that values like `check_time` cannot be specified in config?

Reply:
Yeah. Here's what I did
```
  required_dns_updates {
    check_time = "12345"
  }
```
And here's the error I got
`Can't configure a value for "required_dns_updates": its value will be decided automatically based on the result of applying this configuration.`

Reply:
👍 thanks for checking!